CONTRIBUTING.md

```diff
@@ -1,4 +1,189 @@
-## How to contribute?
+# Contributing to [The Algorithms - Julia](https://github.com/TheAlgorithms/Julia) (WIP)
+
+This document describes how to become a contributor to this repository. This will help you add new algorithms, correct issues and adding new fixes.
```

Comment:
```suggestion
We're glad you want to contribute to the Julia repository! This document describes how to become a contributor to this repository. This will help you add new algorithms, correct issues, and adding new fixes.
```